src/txn/txn_rollback_to_stable.c

```diff
@@ -63,8 +63,8 @@ __txn_rollback_to_stable_lookaside_fixup(WT_SESSION_IMPL *session)
 		if (__bit_test(conn->stable_rollback_bitstring, las_id))
 			continue;
 
```

Comment:
Do we have to do anything special here to remove prepared updates from the las file ?

Reply:
we don't do anything special for prepare updates, when the entries of a page are removed from las, it includes committed updates as well as prepared updates.

Reply:
I think you are right.